Extensions/CodeGeneration/CodeGenerationDisplayDriver.cs

```diff
@@ -13,10 +13,14 @@ namespace Lombiq.HelpfulExtensions.Extensions.CodeGeneration
 {
     public class CodeGenerationDisplayDriver : ContentTypeDefinitionDisplayDriver
     {
+        private const string True = "true";
+        private const string False = "false";

```

Comment:
Ez miért jobb így?

Reply:
Egyrészt ez compile time constant míg a `value.ToString().ToLowerInvariant()` az runtime átalakítással jár. Másrészt meg nem okoz problémát a CA1308. A DocuSignService-ben is ezt csináltam régebben.

Reply:
De miért van szükség constokra sima stringek helyett?

Reply:
Csak hogy ne legyen a kód belsejében magic string. De itt az egész osztály az mert code generation. Akkor nem szükséges.

Reply:
Meg ez annyira nem magic szerintem :).